lib/modules/web3/contracts/wagmi-helpers.ts

```diff
@@ -16,3 +17,16 @@ export function isValidUserAddress(userAddress?: Address) {
   if (userAddress === noUserAddress) return false
   return true
 }
+
+/*
+  Returns timeout to be used in useWaitForTransactionReceipt options
+  By default all react queries retry 3 times
+  More info: https://tanstack.com/query/v5/docs/framework/react/guides/query-retries
+ */
+export function getWaitForReceiptTimeout(chainId: number) {
+  // In polygon there will be 3 retries of 25 seconds until we throw the timeout error
+  if (chainId === polygon.id) return 25_000
```

Comment:
We have helpers to make this more readable.
```suggestion
  if (chainId === polygon.id) return secs(25).toMs()
```